src/ACEtk/PhotoatomicTable.hpp

```diff
@@ -17,23 +20,26 @@ namespace ACEtk {
  *  @class
  *  @brief The photoatomic ACE table
  *
- *  The PhotoAtomicTable class represents the ACE table for the photoatomic data.
+ *  The PhotoatomicTable class represents the ACE table for the photoatomic data.
  */
-class PhotoAtomicTable : protected Table {
+class PhotoatomicTable : protected Table {
 
   /* fields */
   block::ESZG eszg_;
   block::JINC jinc_;
   block::JCOH jcoh_;
+  block::JFLO jflo_;
```

Comment:
Technically, JFLO, EPS, and SWD are optional, but I'm not worried much about that.

Edit: I see, they're just length zero if the NXS values say they're not there.

Reply:
I've probably replaced these with optionals later anyway.